### PR TITLE
added start and endtime

### PR DESCRIPTION
**Pull request by hbee-ac**
added endtime and time column to the bookings table, modified the view, bookings model, booking livewire

### CODE REVIEW

File: DeskIt-Hot-Desk-Booking-System/database/migrations/2024_05_14_122441_add_booking_time_to_bookings_table.php
Comment (Saiyantist) [inline DeskIt-Hot-Desk-Booking-System/database/migrations/2024_05_14_122441_add_booking_time_to_bookings_table.php:1]: bakit pala gumawa ng new migration file?

Comment (hbee-ac) [reply DeskIt-Hot-Desk-Booking-System/database/migrations/2024_05_14_122441_add_booking_time_to_bookings_table.php:1]: > bakit pala gumawa ng new migration file?

kasi kapag nakapag migrate ka na nung una need mo gumawa or magdagdag ng column sa table na na-migrate mo then migrate mo lang yon para mapunta sa db mo. kapag kasi ni-edit ko lang and nimigrate ko nakalagay nothing to migrate raw

Comment (hbee-ac) [reply DeskIt-Hot-Desk-Booking-System/database/migrations/2024_05_14_122441_add_booking_time_to_bookings_table.php:1]: if may iba pa namang paraan para di na gumawa ng new migration, I can work on that if needed.

Comment (Saiyantist) [reply DeskIt-Hot-Desk-Booking-System/database/migrations/2024_05_14_122441_add_booking_time_to_bookings_table.php:1]: na try mo ba mag `php artisan migrate --fresh` ?

Comment (hbee-ac) [reply DeskIt-Hot-Desk-Booking-System/database/migrations/2024_05_14_122441_add_booking_time_to_bookings_table.php:1]: hindi pa, lemme change it. react in this message so I can delete this merge request

**Review by Saiyantist — APPROVED** [review]
ok

**Pull request merged**